support-frontend/assets/components/ticker/contributionTicker.jsx

```diff
@@ -179,7 +179,7 @@ export default class ContributionTicker extends Component<PropTypes, StateTypes>
       https://davidwalsh.name/translate3d
       *************************************************************** */
     const progressBarAnimation = `translate3d(${percentageToTranslate(this.dataFromServer.totalContributed || 0, this.dataFromServer.goal || 0, this.tickerType)}%, 0, 0)`;
-    const readyToRender = (this.state && this.state.count && this.state.count !== 0);
+    const readyToRender = (this.state && !Number.isNaN(this.state.count) && this.state.count > -1);
```

Comment:
could be nicer for this check to happen when fetching the data from the server. feels a bit like sanity checks that are really about that data are spreading into other bits of logic. maybe if we do the `isNaN` checks in `getInitialTickerValues` and throw an error if we don't have real numbers? then maybe `readyToRender` could just depend on `this.dataFromServer`?

(that may not actually simplify things in practice but just a thought....)